app/views/admin/organizations/edit.html.haml

```diff
@@ -1,45 +1,42 @@
-%h1.padded
+%h1.mt-4
   Edit
   = link_to @organization.name, admin_organization_url(@organization)
 
-.form.form-horizontal
-  = form_for [:admin, @organization] do |f|
-    = render partial: 'form', locals: { f: f }
 
-    .nested-form
+= form_for [:admin, @organization] do |f|
+  = render partial: 'form', locals: { f: f }
+  .row.mt-4
+    .col-md-6.mb-4
+      %fieldset
+        #admin-locations-fields
+          = f.fields_for :locations do |builder|
+            = render '/admin/organizations/location_fields', f: builder
+          = revised_link_to_add_fields "Add a location", f, :locations, class_name: "btn btn-primary"
+
+    .col-md-4
+      %a.btn.btn-primary{"aria-controls" => "new_bike_notification", "aria-expanded" => "false", "data-toggle" => "collapse", :href => "#new_bike_notification"}
+        New bike notification
+      #new_bike_notification.collapse
+        .form-group
+        = f.label :new_bike_notification
+        = f.text_area :new_bike_notification, class: "form-control"
+    .col-md-2
       %p
         %strong
           Locations
-      %p#us-country-code.other-value
+      %p#us-country-code
         = Country.united_states.id
-      %fieldset
-        #admin-locations-fields.horizontal-form-list
-          = f.fields_for :locations do |builder|
-            = render '/admin/organizations/location_fields', f: builder
-          = link_to_add_fields "Add a location", f, :locations
-
-    .control-group.actions.big-action.padded
-      = f.submit "Update", class: 'button-green big-action'
-
-    #bike-notification.collapse
-      .control-group
-        .control-label
-          = f.label :new_bike_notification
-        .controls
-          = f.text_area :new_bike_notification
-
-      - if @organization.new_bike_notification.present?
-        .new-notification-container
-          .header-content
-            %img{src: "https://files.bikeindex.org/bikeindex-white-tilt.png"}
 
-          .new-notification
-            = @organization.new_bike_notification.html_safe
+  - if @organization.new_bike_notification.present?
```

Comment:
If the organization has a new bike notification (and a location), it is entirely separated from the form where they enter the new bike notification. They should be together.

Reply:
Actually, as discussed in https://github.com/bikeindex/bike_index/issues/705 - this should be removed from this form (and in a separate PR, removed from Bike Index)